### PR TITLE
fix(AttentionBox): general improvements + allow ellipsis for overflowin attention box

### PR DESCRIPTION
**Pull request by YossiSaadi**
https://monday.monday.com/boards/3532714909/pulses/4183198730

- remove redundant div when no title: no need to render the wrapper flex div in case there is no title (if there's no title, now a div is rendered without content in it)
- add extra padding-right only if attention box is compact and dismissable
- make attention box stretched to container
- allow ellipsis for overflowing attention box

### CODE REVIEW

File: src/components/AttentionBox/AttentionBox.module.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@
   position: relative;
   padding: var(--spacing-medium);
   border-radius: var(--border-radius-medium);
-  width: fit-content;
+  width: 100%;
```
Comment (hadasfa) [inline src/components/AttentionBox/AttentionBox.module.scss:9]: dangerous a bit (breaking change)

**Review by hadasfa — COMMENTED** [review]
let's find a way to make it work without change the default width

File: src/components/AttentionBox/AttentionBox.module.scss (hunk 1 above)
Comment (YossiSaadi) [reply src/components/AttentionBox/AttentionBox.module.scss:9]: but we said that we are intentionally want it 👀

Comment (YossiSaadi) [reply src/components/AttentionBox/AttentionBox.module.scss:9]: will try to solve with fit-content 👍🏼

**Comment by YossiSaadi** [comment]
@hadasfa it seems to work ok with max-width: 100% and width of fit-content.
We're keeping the same behavior of fitting to content in case it does not fit to 100% of container, it will shrink to fit-content

File: src/components/AttentionBox/AttentionBox.module.scss [hunk 2]
```diff
@@ -7,6 +7,7 @@
   padding: var(--spacing-medium);
   border-radius: var(--border-radius-medium);
   width: fit-content;
+  max-width: 100%;
```
Comment (hadasfa) [inline src/components/AttentionBox/AttentionBox.module.scss:10]: Great solution!

**Review by hadasfa — APPROVED** [review]
in general looking good, just want to make sure how it behave in chromatic before merge

**Pull request merged**